components/brave_new_tab_ui/containers/newTab/settings/colorChooser.tsx

```diff
@@ -5,29 +5,47 @@
 
 import * as React from 'react'
 
-import { StyledCustomBackgroundSettings } from '../../../components/default'
+import {
+  SettingsRow,
+  SettingsText,
+  StyledCustomBackgroundSettings
+} from '../../../components/default'
 import NavigateBack from '../../../components/default/settings/navigateBack'
 import ColorBackgroundOption from './colorBackgroundOption'
+import { Toggle } from '../../../components/toggle'
+
+import { getLocale } from '../../../../common/locale'
 
 interface Props {
   title: string
   values: string[]
   currentValue?: string
-  onSelectValue: (value: string) => void
+  usingRandomColor: boolean
+  onSelectValue: (value: string, useRandomColor: boolean) => void
   onBack: () => void
+  onToggleRandomColor: (on: boolean) => void
 }
 
-function ColorChooser ({ title, values, onBack, onSelectValue, currentValue }: Props) {
+function ColorChooser ({ title, values, onBack, onSelectValue, currentValue, usingRandomColor, onToggleRandomColor }: Props) {
   const containerEl = React.useRef<HTMLDivElement>(null)
   React.useEffect(() => {
     containerEl.current?.scrollIntoView(true)
   }, [])
 
+  console.log(usingRandomColor)
```

Comment:
Nit: remove

Reply:
Oh, no! sorry for this again :( I do this all the time. I need a new habit to stop this.